api/test/integration/env/configurations/webhook/manager/config/api/configuration/api.yaml

```diff
@@ -0,0 +1,44 @@
+host: 0.0.0.0
+port: 55000
+
+# Advanced configuration
+
+https:
+ enabled: yes
+ key: "server.key"
+ cert: "server.crt"
+ use_ca: False
+ ca: "ca.crt"
+
+# Logging configuration
+# Values for API log level: disabled, info, warning, error, debug, debug2 (each level includes the previous level).
+logs:
+ level: "info"
+ format: "plain"
+
+# Cross-origin resource sharing: https://github.com/aio-libs/aiohttp-cors#usage
+cors:
+ enabled: no
+ source_route: "*"
+ expose_headers: "*"
+ allow_headers: "*"
+ allow_credentials: no
+
+# Cache (time in seconds)
+cache:
+  enabled: yes
+  time: 0.750
+
+# Access parameters
+access:
+  max_login_attempts: 10000
+  max_request_per_minute: 20
+
+# Drop privileges (Run as wazuh user)
+drop_privileges: yes
+
+# Enable features under development
+experimental_features: yes
+
+# Set the maximun size of the bulk for /events endpoint
+events_bulk_max_size: 5
```

Comment:
What is this? I don't remember we accepted a design where the bulk size would be configurable.

Reply:
Ok, it was a misunderstanding on my side. I'll remove it and use a constant value for that.